src/handler/assets.rs

```diff
@@ -13,7 +13,7 @@ const ONE_YEAR: Duration = Duration::from_secs(60 * 60 * 24 * 365);
 
 pub fn static_assets(config: &ApiConfig) -> impl Handler {
     ReactApp {
-        handler: static_compiled!("$OUT_DIR").with_index_file("index.html"),
+        handler: static_compiled!("$ASSET_DIR").with_index_file("index.html"),
```

Comment:
Does this work? I can't remember how I implemented this and whether $OUT_DIR is special

Reply:
Yes, this reads from the correct environment variable inside a proc macro

Reply:
sweet, I don't remember writing that but it's even tested https://github.com/trillium-rs/trillium/blob/main/static-compiled-macros/src/lib.rs#L294-L311 -- I think before I forked `include_dir` it just special cases a few environment variables